src/AddTodoForm.js

```diff
@@ -1,23 +1,32 @@
 import React from "react";
+import { useState } from "react";
 
-const AddToDoForm = (props) => {
+const AddToDoForm = ({ onAddTodo }) => {
+	const [todoTitle, setTodoTitle] = useState("");
 
+	const handleTitleChange = (event) => {
+		let newTodoTitle = event.target.value;
+		setTodoTitle(newTodoTitle);
+	};
 	const handleAddToDo = (event) => {
 		event.preventDefault();
-		let todoTitle = event.target.title.value;
-		console.log(todoTitle);
-		props.onAddTodo(todoTitle);
-		event.target.reset();
+		onAddTodo({ title: todoTitle, id: Date.now });
+		setTodoTitle([]);
```

Comment:
Everything looks good! Just make sure to setTodoTitle to an empty string per the directions not an empty array. Otherwise great job on this assignment!